Cannot install using make

Hi, I'm trying to install shen-scheme from the binary running the command `make` and I am getting this error

```
echo "Building Chez..."
Building Chez...
cd _build/chez/csv10.0.0 && ./configure --threads && make
Configuring for ta6le, and will create boot files via pb
make[1] : on entre dans le répertoire « /home/cactus/Documents/coding/shen-scheme/_build/chez/csv10.0.0 »
mkdir -p bin
gcc -m64 -msse2 -O2 -DZUO_LIB_PATH='"'"../zuo/lib"'"' -o bin/zuo ./zuo/zuo.c
./zuo/zuo.c: Dans la fonction « zuo_init_signal_handler »:
./zuo/zuo.c:5585:17: erreur: affectation à « __sighandler_t » {alias « void (*)(int) »} depuis le type pointeur « void (*)(void) » qui est incompatible [-Wincompatible-pointer-types]
 5585 |   sa.sa_handler = zuo_signal_received;
      |                 ^
./zuo/zuo.c:5565:13: note: « zuo_signal_received » déclaré ici
 5565 | static void zuo_signal_received() {
      |             ^~~~~~~~~~~~~~~~~~~
Dans le fichier inclus depuis /usr/include/sys/wait.h:36,
                 depuis ./zuo/zuo.c:23:
/usr/include/signal.h:72:16: note: « __sighandler_t » déclaré ici
   72 | typedef void (*__sighandler_t) (int);
      |                ^~~~~~~~~~~~~~
make[1]: *** [Makefile:152: bin/zuo] Error 1
make[1] : on quitte le répertoire « /home/cactus/Documents/coding/shen-scheme/_build/chez/csv10.0.0 »
make: *** [Makefile:108: _build/chez/csv10.0.0/ta6le/boot/ta6le/libkernel.a] Error 2
```

I wonder how can I if someone know how to fix this?

I was getting the same error and then I modified the makefile like in #32 

and I'm getting (translated)

> make: *** No rule to make target ‘shen-scheme.scm’, needed by ‘_build/lib/shen-scheme/shen.boot’. Stop.

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -20,7 +20,7 @@ jobs:
       - uses: actions/checkout@v2
       - name: Chez cache
         id: shen-scheme-chez10-cache
-        uses: actions/cache@v2
+        uses: actions/cache@v3
         with:
           path: _build/chez
           key: ${{ matrix.os }}-chez-${{ env.CHEZ_SCHEME_VERSION }}

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -11,7 +11,7 @@ jobs:
     name: Build and test
     runs-on: ${{ matrix.os }}-latest
     env:
-      CHEZ_SCHEME_VERSION: '10.0.0'
+      CHEZ_SCHEME_VERSION: '10.2.0'
     strategy:
       matrix:
         os: [macos, ubuntu, windows]

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -31,7 +31,7 @@ jobs:
   build:
     needs: create_release
     env:
-      CHEZ_SCHEME_VERSION: '10.0.0'
+      CHEZ_SCHEME_VERSION: '10.2.0'
     strategy:
       matrix:
         config:

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -68,7 +68,7 @@ jobs:
         uses: actions/checkout@v2
       - name: Chez cache
         id: shen-scheme-chez-cache
-        uses: actions/cache@v2
+        uses: actions/cache@v3
         with:
           path: _build/chez
           key: shen-scheme-chez-cache-${{ matrix.config.os }}-${{ matrix.config.arch }}-${{ env.CHEZ_SCHEME_VERSION }}

--- a/Makefile
+++ b/Makefile
@@ -56,7 +56,7 @@ ifeq ($(os), linux)
 endif
 
 shenversion ?= 39.0
-csversion ?= 10.0.0
+csversion ?= 10.2.0
 build_dir ?= _build
 chez_build_dir ?= $(build_dir)$(S)chez
 csdir ?= $(chez_build_dir)$(S)csv$(csversion)

--- a/Makefile
+++ b/Makefile
@@ -108,7 +108,7 @@ $(cskernel): $(csdir)
 ifeq ($(os), windows)
 	cmd.exe /C 'cd $(csdir) && build.bat ta6nt'
 else
-	cd $(csdir) && ./configure --threads && make
+	cd $(csdir) && ./configure --threads --disable-curses --disable-iconv --disable-x11 && make
 endif
 
 .PHONY: chez_kernel

--- a/Makefile
+++ b/Makefile
@@ -27,7 +27,7 @@ ifeq ($(os), windows)
 	arext = .lib
 	binext = .exe
 	archiveext = .zip
-	cskernelname = csv1000mt
+	cskernelname = csv1020mt
 	lz4dirname = lz4mts$(S)lib
 	lz4libname = liblz4
 	zlibdirname = zlibmts

